.ci/integration_test.groovy

```diff
@@ -235,6 +291,19 @@ def call(ghprbActualCommit, ghprbCommentBody, ghprbPullId, ghprbPullTitle, ghprb
                             sh "cat pd.log"
                             sh "cat tikv.log"
                             sh "cat tidb.log"
+                            if (TEST_TIFLASH != "false") {
+                                try {
+                                    sh "cat tiflash_cmd_line.log"
+                                    sh "ls -l tiflash/"
+                                    sh "cat tiflash/tiflash_error.log"
+                                    sh "cat tiflash/tiflash.log"
+                                    sh "cat tiflash/tiflash_cluster_manager.log"
+                                    sh "cat tiflash/tiflash_tikv.log"
+                                    sh "echo 'done'"
```

Comment:
if `tiflash/tiflash_error.log` does not exist, the following log will not be printed.